File: leet_easy_c++/leet_058/leet_058.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int j = s.size() - 1;
        if(j<0) return 0;
        while(j>0 && *(s.begin() + j) == ' ') j--;
        if(j==0 && *(s.begin()) == ' ') return 0;
        int i = j;
        while(i>0 && *(s.begin() + i) != ' ') i--;
        if(*(s.begin()+i) == ' ') return j - i;
        return j - i + 1;
    }
};

File: leet_easy_python/leet_058/leet_058.py
class Solution(object):
    def lengthOfLastWord(self, s):
        if not s: return 0
        j = len(s) - 1
        cnt = 0
        while j>=0 and s[j] == ' ': j-=1
        while j>=0 and s[j] != ' ': 
            j-=1
            cnt+=1
        return cnt
        
        """
        :type s: str
        :rtype: int
        """